README.md

```diff
@@ -18,17 +18,17 @@ This Project serves the backend APIs required for [Real Dev Squad](https://reald
 
 ## Running the Project
 ```shell
-$ npm install
-$ npm start
+npm install
+npm start
```

Comment:
Let's remove `npm start` from here? This command is more suited for our heroku server than our development servers, most people use this command, make some changes and keep wondering why changes are not being reflected (this command uses `node` and not `nodemon`)